aries-site/src/examples/templates/list-views/ListIconIdentifierExample.js

```diff
@@ -0,0 +1,64 @@
+import React from 'react';
+import { Box, List, Menu, Text } from 'grommet';
+import {
+  Console,
+  More,
+  PersonalComputer,
+  Desktop,
+  User,
+  System,
+} from 'grommet-icons';
+
+const data = [
+  {
+    name: 'Title 1',
+    option: 'Active',
+    icon: <PersonalComputer />,
+  },
+  {
+    name: 'Title 2',
+    option: 'Inactive',
+    icon: <User />,
+  },
+  {
+    name: 'Title 3',
+    option: 'Active',
+    icon: <Desktop />,
+  },
+  {
+    name: 'Title 4',
+    option: 'Active',
+    icon: <System />,
+  },
+  {
+    name: 'Title 5',
+    option: 'Inactive',
+    icon: <Console />,
+  },
+];
+
+export const ListIconIdentifierExample = () => {
```

Comment:
I am seeing the UserAgent focus color come through. Is that what we want?
![Screen Shot 2020-07-06 at 1 28 28 PM](https://user-images.githubusercontent.com/1756948/86632532-c0d91400-bf8c-11ea-9cf2-60b3c94210cc.png)

Reply:
I am not sure we want this. It is occurring in the generic grommet storybook as well, so may be result of implementation and a fix that would happen on the grommet side? https://storybook.grommet.io/?path=/story/list--onclickitem

Reply:
Agree. We should create an issue in Design System for prioritization and tracking purposes and link it to a corresponding issue in Grommet.

Reply:
Great, I will open both of those tickets.

Reply:
Being tracked on Design System: https://github.com/hpe-design/design-system/issues/965
and on grommet: https://github.com/grommet/grommet/issues/4278